src/lib/autoinstall/common_helpers.rb

```diff
@@ -0,0 +1,22 @@
+Yast.import "Linuxrc"
+Yast.import "Stage"
+
+module Y2Autoinstallation
+  # This module defines some methods that are used by different classes
+  module CommonHelpers
```

Comment:
To be honest I do not like much that generic CommonHelpers. It tends to overgrow over time. What about having specific name like EFIDetection? And maybe it can be used in future also in bootloader. That common helpers usually has issue that reuse it elsewhere is often hard.

Reply:
In fact I started adding a specific class with method `EFIDetector.in_use?` but as it only provided a class method it smelled to  a Module instead of a class and that is why I added this one, as could be useful for adding some small method shared by others, but as you comment, we tend to add there things that belongs to different modules and at the end you have a lot of methods that you do not need.